src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -131,6 +131,15 @@
                 "fraction_owned",
                 "ownership_record_type",
             ],
+            # These columns identify unique records, but can't be used as a PK because
+            # some of the utility_id_eia values are NA.
+            # "primary_key": [
+            #    "plant_id_eia",
+            #    "generator_id",
+            #    "utility_id_eia",
+            #    "report_date",
+            #    "ownership_record_type",
+            # ],
```

Comment:
can you add this info into `additional_primary_key_text` in the `? (go look at `out_ferc714__respondents_with_fips` for an example of where to put this and we are actively writing up better docs about how to add to the description)